build/yarn.lock

```diff
@@ -2627,14 +2627,14 @@ mimic-response@^3.1.0:
   resolved "https://registry.npmjs.org/mimic-response/-/mimic-response-3.1.0.tgz"
   integrity sha512-z0yWI+4FDrrweS8Zmt4Ej5HdJmky15+L2e6Wgn3+iK5fWzb6T3fhNFq2+MeTRb064c6Wr4N/wv0DzQTjNzHNGQ==
 
-minimatch@3.0.4, minimatch@^3.0.4:
-  version "3.0.4"
-  resolved "https://registry.npmjs.org/minimatch/-/minimatch-3.0.4.tgz"
-  integrity sha512-yJHVQEhyqPLUTgt9B83PXu6W3rx4MvvHvSUvToogpwoGDOUQ+yDrR0HRot+yOCdCO7u4hX3pWft6kWBBcqh0UA==
+minimatch@3.0.5:
```

Comment:
I don't think this is going to be valid - dir-compare requires a pinned version of 3.0.4 : 

```
dir-compare@^2.4.0:
  version "2.4.0"
  resolved "
[https://registry.npmjs.org/dir-compare/-/dir-compare-2.4.0.tgz"](https://registry.npmjs.org/dir-compare/-/dir-compare-2.4.0.tgz%22)

  integrity sha512-l9hmu8x/rjVC9Z2zmGzkhOEowZvW7pmYws5CWHutg8u1JgvsKWMx7Q/UODeu4djLZ4FgW5besw5yvMQnBHzuCA==
  dependencies:
    buffer-equal "1.0.0"
    colors "1.0.3"
    commander "2.9.0"
    minimatch "3.0.4"
```

so the only way to update that is to update dir-compare itself, or use resolutions in the package.json.